test/e2e/storage/drivers/csi.go

```diff
@@ -75,14 +75,14 @@ type hostpathCSIDriver struct {
 func initHostPathCSIDriver(name string, capabilities map[testsuites.Capability]bool, volumeAttributes []map[string]string, manifests ...string) testsuites.TestDriver {
 	return &hostpathCSIDriver{
 		driverInfo: testsuites.DriverInfo{
-			Name:        name,
-			FeatureTag:  "",
-			MaxFileSize: testpatterns.FileSizeMedium,
+			Name:       name,
+			FeatureTag: "",
 			SupportedFsType: sets.NewString(
 				"", // Default fsType
 			),
 			SupportedSizeRange: volume.SizeRange{
 				Min: "1Mi",
+				Max: strconv.FormatInt(testpatterns.FileSizeMedium, 10),
```

Comment:
I think hostPath driver should not have a max size limitation

Reply:
this was translated from original config:
```
                       MaxFileSize: testpatterns.FileSizeMedium,
```